src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -157,6 +157,44 @@ else if (computer_battleship.scan(coor)){
         scanRequest = true;
     }
 
+
+
+    public void sinkComputerCivShip (CivShip ship) {
```

Comment:
Good job making a descriptive function name here. Someone was paying attention in class yesterday